### PR TITLE
fix(po): Update spanish.po

### PR DESCRIPTION
**Pull request by kolmo7**
It needs to be reviewed again to see if they are all translated strings and once the language file is available for use with version 1.3.0 rc.1

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/895)

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 639992c4270e8456ca3423c45a07b6424c8d4e84
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/spanish.po [hunk 1]
```diff
@@ -506,14 +506,17 @@ msgstr "&Cancelar "
...
+msgstr "Transacción incorrecta"
 
 #: src/assetdialog.cpp:405
 msgid ""
 "Asset Account not found.\n"
 "\n"
 "Would you want to create one?"
 msgstr ""
+"Cuenta de activos no encontrada.\n"
+"\n!
```
Comment (stef145g) [inline po/spanish.po]: Line 518/519 preventing build checks to proceed

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 639992c4270e8456ca3423c45a07b6424c8d4e84
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 546df596b624f255b28083d34c81d0fdcee3acea
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/spanish.po (hunk 1 above)
Comment (kolmo7) [inline po/spanish.po]: Changed "\n! for "\n"

**Comment by vomikan** [comment]
It should be 3 lines in the translation. There are only 2.

**Comment by kolmo7** [comment]
Vomikan, in the spanish translation file there are three lines.

**Pull request merged**